tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -126,20 +130,24 @@ sub run {
     if (is_s390x) {
         #upload s390x_guest_upgrade_test.log
         upload_asset("/tmp/s390x_guest_upgrade_test.log", 1, 1);
-        script_run "rm -rf /tmp/s390x_guest_upgrade_test.log";
+        script_run "rm -rf /tmp/s390x_guest_upgrade_test.log", %args;
```

Comment:
Why 'do not die' on timeout here. I think this should die on timeout. Or say 'die on timeout' does not make it worse, maybe better for warning purpose.